# Generated for NaoQi method ALVisionRecognition.load
# load the database.ar1, ar2, ar3, ar4 and ar5 files that have to be on the robot in /home/nao/naoqi/share/naoqi/vision/visionrecognition/current.

---


